site/src/api/api.ts

```diff
@@ -32,6 +32,8 @@ import type {
 } from "./typesGenerated";
 import * as TypesGen from "./typesGenerated";
 
+const API_PREFIX = "/api/v2";
```

Comment:
Yikes, I didn't realise we hardcoded the base for all these.
Thanks for cleaning the campsite as you go ❤️